README.md

```diff
@@ -1,44 +1,63 @@
-# Bendel-Bot
-## Authors
-
-* Raymond Zheng
-* Hunter Spack
-* Drew Marcotte
-* Joe Marchesini
-## Bot Set Up
-
-1. Run the command "pip install -U discord.py" in your command line.
-2. Run the command "pip install -U python-dotenv" in your command line.
-3. Create a .env file with a token provided by one of the developers and put this text in it in this format: DISCORD_TOKEN=PROVIDED_TOKEN
-4. Navigate to the folder containing the bot in your command line.
-5. Run the command "python bot.py" in your command line to run the bot.
-
-If you are just running the bot on your computer, terminate using the Control key and C hotkey, otherwise terminate like you would any other program.
-
-## Features
-**Welcome Messages**
-- Bendel-Bot gives new server members a warm welcome with any one of 15+ welcome messages
-
-**React Roles**
-- React to a message with an emoji to assign yourself a role.
-- Use !roleset (Role Name) (msg id to react to) (Emoji) to configure.
-- Ex. !roleset dev 824395559791755374 :thumbsup:
-
-## Planned Bot Features
-**Meme Delivery**
-- Bendel-Bot will share a random meme on command from their personal stash 
-
-**User Currency Balance**
-- Each member of the server will be able to earn Bendel-Bucks(BBs) and have access to their Bendal-Balance where they can view how many BBs they have.
-- Members will be able to spend Bendel-Bucks on a number of different activities
-
-**Slot Machine**
-- One of those planned activities is Bendel-Bot's slot machine where server members will be able to spend BBs to give it a spin to try and win more BBs
-
-**Music**
-- Plays music through the bot using the Spotify API
-- Members can possibly use BBs to push their song to the front of the queue
-
-## Features that are less certain to be featured
-**Bendel-Buck Leaderboard**
-- Run a command to see the top Bendel-Buck earners on the server
+# Bendel-Bot
+## Authors
+
+* Raymond Zheng
+* Hunter Spack
+* Drew Marcotte
+* Joe Marchesini
+
+## Bot Set Up
+1. Open the bot folder as a project in PyCharm and then set up a virtual environment for your bot.
+2. Run the command "pip install -U discord.py" in your command line.
+3. Run the command "pip install -U python-dotenv" in your command line.
+4. Run the command "pip install --upgrade youtube_dl" in your command line.
+5. Run the command "pip install PyNaCl" in your command line.
```

Comment:
Update to reflect my new format of readme listing the pip installs in its own section